backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -684,6 +735,27 @@ export const isContributionValid = (
   return true
 }
 
+function isStartEndDateValid(
+  startDate: string | null | undefined,
+  endDate: string | null | undefined,
+) {
+  if (!startDate) {
+    logger.error('Start-Date is not initialized. A Start-Date must be set!')
+    throw new Error('Start-Date is not initialized. A Start-Date must be set!')
+  }
+
+  if (!endDate) {
```

Comment:
If we require start and end-date, we should update our schema. There are both optional there:
```
  @Field(() => String, { nullable: true })
  validFrom?: string | null

  @Field(() => String, { nullable: true })
  validTo?: string | null
```
in `backend/src/graphql/arg/ContributionLinkArgs.ts`